src/runtime/code.cpp

```diff
@@ -133,9 +133,9 @@ extern "C" PyCodeObject* PyCode_New(int argcount, int nlocals, int stacksize, in
                                     PyObject* lnotab) noexcept {
     // Check if this is a dummy code object like PyCode_NewEmpty generates.
     // Because we currently support dummy ones only.
-    bool is_dummy = argcount == 0 && nlocals == 0 && stacksize == 0 && flags == 0;
+    bool is_dummy = stacksize == 0;
     is_dummy = is_dummy && code == EmptyString && lnotab == EmptyString;
-    for (auto&& var : { consts, names, varnames, freevars, cellvars })
+    for (auto&& var : { consts, names, freevars, cellvars })
```

Comment:
While I don't really like that this just pretends that we support this stuff I'm okay with this workaround for now because I think we will notice if code tries to retrieve the fields because it will crash because `BoxedCode::f` will be NULL.

Could you please update the comment because it does not match the check anymore.
I would suggest to leave the whole `is_dummy` computation unchanged and instead make the assert less strict by allowing the fields you enabled but adding a comment which says that it's not implemented for now.